floodviz/static/js/map.js

```diff
@@ -236,6 +232,34 @@
 				svg.select('#map-select-box').remove();
 			};
 
+			var make_sites_bland = function () {
+				Object.keys(state.gages).forEach(function (key) {
+					const g = state.gages[key];
+					var style = 'gage-point';
+					if (g.accent) {
+						style += '-accent'
+					}
+					if (g.bland) {
+						style += '-bland';
+					}
+					else {
+						g.bland = true;
+					}
+					d3.select('#map' + key).attr('class', style);
```

Comment:
What this does is make all but one of the sites bland. When you moseover a site `123`, `state.gages[123].bland` is set to `false` so that it will be exempted here. I set it back to `true` here to make sure it can be set to bland next time. I did it this way because I wanted to make it easy to have multiple sites exempted if we want to add that in the future, but I'd be open to changing it. It would probably be easier for now to just pass a single site ID that we want to exempt.

There are two 'bland' styles `gage-point-bland` and `gage-point-accent-bland` so that the sites displayed on the hydrograph are still distinguished from the sites that aren't. This won't change whether or not a gage is accented, it just checks which bland style it should apply to each.